WORKSPACE

```diff
@@ -0,0 +1,12 @@
+git_repository(
+    name = "build_bazel_rules_apple",
+    remote = "https://github.com/jerrymarino/rules_apple.git",
+    tag = "0.4.0.2-pinterest",
```

Comment:
Latest rules apple has our patch so we can just point to 0.5.0

Reply:
Done